src/main/java/com/nexmo/quickstart/voice/OutboundTextToSpeech.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2011-2017 Nexmo Inc
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package com.nexmo.quickstart.voice;
+
+import com.nexmo.client.NexmoClient;
+import com.nexmo.client.auth.AuthMethod;
+import com.nexmo.client.auth.JWTAuthMethod;
+import com.nexmo.client.voice.Call;
+
+import java.nio.file.FileSystems;
+
+public class OutboundTextToSpeech {
+    public static void main(String[] args) throws Exception {
+        String APPLICATION_ID = envVar("APPLICATION_ID");
+        String PRIVATE_KEY_PATH = envVar("PRIVATE_KEY_PATH");
+        String FROM_NUMBER = envVar("FROM_NUMBER");
+        String TO_NUMBER = envVar("TO_NUMBER");
+
+
+        AuthMethod auth = new JWTAuthMethod(
+                APPLICATION_ID,
+                FileSystems.getDefault().getPath(PRIVATE_KEY_PATH)
```

Comment:
Oh, the reason I ask is that the other quick start examples use `PRIVATE_KEY` and it _may_ be worth being consistent with that.